ci_fuel_opnfv/ci_pipeline.sh

```diff
@@ -428,11 +432,49 @@ function deploy {
     echo "Repo path is ${REPO_PATH}"
 
     if [ $BUILD -eq 1 ]; then
-       echo sudo python ${REPO_PATH}/fuel/deploy/deploy.py ${BUILD_ARTIFACT_STORE}/${BRANCH}/${VERSION}/${ISO} ${DEA} ${DHA}
-       sudo python ${REPO_PATH}/fuel/deploy/deploy.py ${BUILD_ARTIFACT_STORE}/${BRANCH}/${VERSION}/${ISO} ${DEA} ${DHA}
+       ISOFILE=${BUILD_ARTIFACT_STORE}/${BRANCH}/${VERSION}/${ISO}
     else
-       echo sudo python ${REPO_PATH}/fuel/deploy/deploy.py ${LOCAL_ISO} ${DEA} ${DHA}
-       sudo python ${REPO_PATH}/fuel/deploy/deploy.py ${LOCAL_ISO} ${DEA} ${DHA}
+       ISOFILE=${LOCAL_ISO}
+    fi
+
+
+    VERSION=$(7z x -so $ISOFILE version.yaml 2>/dev/null \
+        | grep release | sed 's/.*: "\(...\).*/\1/')
+
+    if [ -n "${VERSION}" ]; then
+        echo "Fuel version is in ISO is ${VERSION}"
+    else
+        echo "Error: Could not retrieve Fuel version from $ISOFILE"
+        exit 1
+    fi
+
+    if [ -d "${SCRIPT_PATH}/config/${VERSION}"  ]; then
+        DEA=${SCRIPT_PATH}/config/${VERSION}/${DEPLOY_CONFIG}/dea.yaml
+        DHA=${SCRIPT_PATH}/config/${VERSION}/${DEPLOY_CONFIG}/dha.yaml
+        if [ ! -f $DEA ]; then
+            echo "Could not find DEA file $DEA"
+            exit 1
+        fi
+
+        if [ ! -f $DHA ]; then
+            echo "Could not find DHA file $DHA"
+            exit 1
+        fi
+
+        # Handle different deployer versions
+        case "${VERSION}" in
+            "6.0")
```

Comment:
Shouldn't this be 6.1? 6.0 Covers all commits from ARNO SR0 and -iso -dea -dha was introduced a little than a week ago. And from yesturday the args are backwards compatible. So if we test against 6.1 A few (less than a handfull) will not deply, but if we test against 6.0 no commits up till end last week will deploy. Or have I got it wrong?